### DISCUSSION TITLE
Organize repositories

### DISCUSSION THREAD

**Discussion by danisoaresl** [community/community#129902] [Repositories] [answered]
### Select Topic Area

General

### Body

Hi, everyone. I would like to know how you organize your repositories. Do you organize them by types of projects or do you just create them randomly as needed?

**Comment by GessycaBorges** [comment]
Eu crio aleatoriamente, nunca tinha parado pra pensar em como organizar.

**Comment by Sworzen1** [comment, marked as answer]
Hello,
it depends on project and organization structure etc.
In my opinion the most important things to configure are:

1. Clear repository naming: Use semantic name.
2. Branch management: Define a clear main branch and use feature branches for development.
3. Issue tracking: Utilize issues to track tasks, bugs, and enhancements.
4. Pull requests: Establish guidelines for code reviews and merging changes.
5. Documentation: Maintain a comprehensive README and project wiki for new contributors etc.
6. Project boards: Organize tasks with project boards (e.g., To Do, In Progress, Done).
7. Access control: Manage permissions and access rights effectively.
8. Regular maintenance: Clean up unused branches and archive or delete repositories as needed.
